src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -218,6 +218,181 @@ void uct_rc_mlx5_iface_common_tag_cleanup(uct_rc_mlx5_iface_common_t *iface,
 #endif
 }
 
+
+#if HAVE_IBV_EXP_DM
+const uint32_t uct_rc_mlx5_dm_key = 1245; /* magic number */
```

Comment:
pls #define it along with UCT_IB_MLX5_WORKER_BF_KEY

Reply:
done